l10n-pocket/en/pocket/add.ftl

```diff
@@ -0,0 +1,50 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev-pocket.allizom.org/about/
+
+pocket-how-to-save = How to Save
+pocket-save-to-pocket = Save to { -brand-name-pocket } from { -brand-name-firefox }
+pocket-click-the-pocket-button = Click the { -brand-name-pocket} button in your { -brand-name-firefox } toolbar to save articles, videos, pages, and more.
+pocket-learn-how = Learn how
+pocket-dont-see-the-pocket = Don't see the { -brand-name-pocket } buttton?
+
+pocket-view-from-any = View from any Device
+# this is a title for a list of different app stores where you can download pocket
+pocket-get-pocket-for = Get { -brand-name-pocket } for:
+
+pocket-apple-app-store = { -brand-name-apple-app-store }
+pocket-mac-app-store = { -brand-name-mac-short } app store
+pocket-google-play-store = { -brand-name-google-play } store
+pocket-amazon-app-store = { -brand-name-amazon } app store
+pocket-windows = { -brand-name-windows }
+pocket-kobo-e-reader = { -brand-name-kobo } e-reader
+pocket-windows-mobile = { -brand-name-windows-mobile }
+pocket-blackberry = { -brand-name-blackberry }
+pocket-more-apps = More Apps
+
+pocket-unlimited-ways = Unlimited Ways to Save
+pocket-save-via-email = Save via e-mail
+
+# Variables
+#   $add_email_link (email link) mailto:add@getpocket.com
+#   $add_email (string) add@getpocket.com
+pocket-email-any-link = email any link to your list by sending it to <a href="{ $add_email_link}" class="dark-inline-link">{ $add_email }</a>
```

Comment:
Suggestion: keep CSS class names out of fluent strings and use a variable instead

E.g. `class="{ $add_email_css_class }"`

Reply:
```suggestion
pocket-email-any-link = Email any link to your list by sending it to <a href="{ $add_email_link}" class="dark-inline-link">{ $add_email }</a>
```